pygmt/tests/test_colorbar.py

```diff
@@ -65,3 +65,53 @@ def test_colorbar_shading_list():
     fig.basemap(region=[0, 10, 0, 10], projection="X15c", frame="a")
     fig.colorbar(cmap="geo", shading=[-0.7, 0.2], frame=True)
     return fig
+
+
+@pytest.mark.mpl_image_compare
+def test_colorbar_frame_parameters():
+    """
+    Create colorbar with arguments for xlabel, ylabel, and annotation, and no
+    argument passed to frame.
+    """
+    fig = Figure()
+    fig.colorbar(
+        cmap="rainbow",
+        position="x0c/0c+w1c/0.5c",
+        xlabel="test-x",
+        ylabel="test-y",
+        annotation=0.25,
+    )
+    return fig
+
+
+@pytest.mark.mpl_image_compare
+def test_colorbar_frame_list_parameters():
+    """
+    Create colorbar with a list passed to frame and an argument passed to
+    xlabel.
+    """
+    fig = Figure()
+    fig.colorbar(
+        cmap="rainbow",
+        position="x0c/0c+w1c/0.5c",
+        frame=["a.25", "y+lylabel"],
+        xlabel="test-x",
+    )
+    return fig
+
+
+@pytest.mark.mpl_image_compare
+def test_colorbar_frame_string_parameters():
+    """
+    Create colorbar with a string passed to frame and arguments passed to
+    xlabel and ylabel.
+    """
+    fig = Figure()
+    fig.colorbar(
+        cmap="rainbow",
+        position="x0c/0c+w1c/0.5c",
+        frame="a.5",
```

Comment:
> I added that your suggestion and changed the parameters to this, which should be the same output as `test_colorbar_frame_parameters`:
> 
> ```
>     fig.colorbar(
>         cmap="rainbow",
>         position="x0c/0c+w1c/0.5c",
>         frame="a.25",
>         xlabel="test-x",
>         ylabel="test-y",
>     )
> ```
> 
> The test then fails; am I missing a step to make the figures match up?

Should be 0.25 to match `test_colorbar_frame_parameters`.

```suggestion
        frame="a0.25",
```